std-bits/table/src/main/java/org/enso/table/data/column/storage/StringStorage.java

```diff
@@ -18,8 +22,8 @@
 
 /** A column storing strings. */
 public final class StringStorage extends SpecializedStorage<String> {
-
   private final TextType type;
+  private Future<Long> countUntrimmed;
```

Comment:
nitpick, feel free to ignore

`countUntrimmed` is a _verb_ - "to count" the untrimmed values - so it is a good name for a function

IMO for a cached value, it is better to use a noun - `untrimmedCount`